REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
Not sure I follow... I'm talking about replacing the calls to `deriv_central` (from the `GSL.jl` library) with a calls to FowardDiff.jl --- thus ending our dependence on `GSL.jl`. Oh, are you saying the ForwardDiff API may not be flexible enough for that?